packages/rum-core/src/domain/view/trackViews.spec.ts

```diff
@@ -317,6 +317,22 @@ describe('view lifecycle', () => {
       expect(getViewCreateCount()).toEqual(1)
     })
   })
+
+  it('should notify VIEW_CREATED before VIEW_CREATED', () => {
+    setupBuilder.build()
+
+    expect(notifySpy.calls.argsFor(0)[0]).toEqual(LifeCycleEventType.BEFORE_VIEW_CREATED)
+    expect(notifySpy.calls.argsFor(1)[0]).toEqual(LifeCycleEventType.VIEW_CREATED)
+  })
+
+  it('should notify BEFORE_VIEW_ENDED before VIEW_ENDED', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏need to be updated as well 🙂